pkg/filepos/position.go

```diff
@@ -39,6 +39,10 @@ func (p *Position) Line() int {
 	return *p.line
 }
 
+func (p *Position) FileString() string {
```

Comment:
one of the rationals for not defining FileString() or similar method was to force consumers to use AsString() or AsCompatString() to provide consistent way of referencing to file location. do we really need to point to a file and not line number?

Reply:
I agree, it feels bad to have a one-off function for this. It's currently only used in a single error message: one where non-empty data values, and an empty schema were provided. That error looks like this:

```
data values were found in data values file(s), but schema (schema.yml) has no values defined
(hint: define matching keys from data values files(s) in the schema, or do not enable the schema feature)
```

Is there a better way to report it? It felt pretty strange to see something like

```
data values were found in data_values.yml:2, but schema.yml:1 has none defined.
(hint: define matching keys from data values files(s) in the schema, or do not enable the schema feature)
```

What lines are we pointing to – the first instance of a data value? The document start of a schema?